src/coffee/renuo_cms_client.coffee

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+class RenuoCmsClient
+
+  constructor: ->
+    @initializeContentBlocks =>
+      $('a.edit').click =>
+        # TODO: Fix this hack!
+        `var editLink = this`
+        @getContentBlockForEdit $(editLink).parent().parent().attr('data-block'), =>
+          $('a.save').click =>
+            `var saveLink = this`
+            newContent = $(saveLink).parent().parent().find('textarea').val()
+            blockID = $(saveLink).parent().parent().attr('data-block')
+            @updateContentBlock blockID, newContent
```

Comment:
`this` => this
this => _this

when it's rewritten in typescript, it will be solved from itself